### ISSUE TITLE
Introduce multy level thresholds

### ISSUE THREAD

**Issue by pavel-odintsov** [pavel-odintsov/fastnetmon#62]
Subject!

It will be useful for different levels of attack mitigation. We can try to drop traffic on firewall up to 1mpps but on 5 mpps we should do RTBH.

**Comment by s4m4n** [comment]
Awesome, when integrating with Exabgp we can consider this to tag different "BGP Communities" for each level of attack!

**Comment by pavel-odintsov** [comment]
Yep! But we still should maintain limit-sets on FastNetMon side manually. As you can see API for FastNetMon is not important but BGP integration could provide really huge benefit.

**Comment by laf** [comment]
@pavel-odintsov if you get some time, can you jump into irc so we can chat about this feature?

**Comment by pavel-odintsov** [comment]
I'm in :) Just started IRC client.
